go/vt/vtorc/logic/vtorc.go

```diff
@@ -73,26 +70,6 @@ func init() {
 	})
 }
 
-// used in several places
-func instancePollSecondsDuration() time.Duration {
-	return time.Duration(config.Config.InstancePollSeconds) * time.Second
-}
-
-// acceptSighupSignal registers for SIGHUP signal from the OS to reload the configuration files.
-func acceptSighupSignal() {
```

Comment:
@GuptaManan100 double-checking: the 3rd party library handles `SIGHUP` reload?

Reply:
SIGHUP is not even required anymore. Viper has a watcher on the file, and if it changes, it automatically reloads the configurations.